### ISSUE TITLE
#warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]

### ISSUE THREAD

**Issue by XVilka** [yuriy-chumak/ol#12]
On Fedora 25:
```
[user@localhost ol]$ make
vm repl <src/boot.lisp >src/boot.c
gcc -std=c99  -O2 -DNDEBUG -s    -DNO_SECCOMP -lasan -fsanitize=address -fno-omit-frame-pointer src/olvm.c src/boot.c -o ol \
   -Xlinker --export-dynamic  -ldl
In file included from /usr/include/stdint.h:25:0,
                 from /usr/lib/gcc/x86_64-redhat-linux/6.3.1/include/stdint.h:9,
                 from src/olvm.c:223:
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^~~~~~~
Ok.
```

**Comment by yuriy-chumak** [comment]
Done in c808e61a259984e21e69549a1c4539942175d06c, but confirmation is required.

**Comment by XVilka** [comment]
No, still the same warning

**Comment by yuriy-chumak** [comment]
Can you provide output of `gcc -dM -E - < /dev/null`?

**Comment by XVilka** [comment]
Added for both Gentoo and Fedora, both have this warning.

[gcc_defines_gentoo.txt](https://github.com/yuriy-chumak/ol/files/921357/gcc_defines_gentoo.txt)
[gcc_defines_fedora.txt](https://github.com/yuriy-chumak/ol/files/921358/gcc_defines_fedora.txt)

**Comment by yuriy-chumak** [comment]
Thank you. Analyzing...

**Comment by yuriy-chumak** [comment]
Fixed in d722740aa3780e244e04d19de346bac426206d1c.
Checked with Fedora 25.
